src/js/shield_defs.js

```diff
@@ -1142,6 +1142,12 @@ export function loadShields() {
     notext: true,
   };
 
+  shields["GLCT"] = {
+    notext: true,
+  };
+
+  shields["GLCT:Loop"] = banneredShield(shields["GLCT"], ["LOOP"]);
+
```

Comment:
Since the loop shield seems to be designated by a brown shield rather than the green one, is it redundant to add the "LOOP" text?

Reply:
I think so; both Loops and Scenic Spurs have a brown shield.

Reply:
You think it's redundant, or you think that we need it because there are both loops and spurs?

Reply:
Sorry, I think it’s helpful to keep the banner in place because there are two kinds of routes that use the same base shield.

Reply:
Sounds good!